grammar.js

```diff
@@ -19,6 +19,7 @@ module.exports = grammar({
       'binary_relation',
       'binary_concat',
       'clause_connective',
+      'pattern_matching',
```

Comment:
These two need to switch right now they are grouping the predicates like this because `LIKE` is given higher precedence than `AND`.

```sql
WHERE a LIKE ('%a' AND a) NOT LIKE '%a'
```

```suggestion
      'pattern_matching',
      'clause_connective',
```

Reply:
Oh, ok. I was not aware that the order matters here.